### DISCUSSION TITLE
Flash Jig Emporia Vue 3

### DISCUSSION THREAD

**Discussion by MichaelWoodc** [emporia-vue-local/esphome#305] [Show and tell]
Hey guys, I didn't want to solder to my board, so I made this: https://www.printables.com/model/979218-emporia-vue-v3-esp-home-flash-jig
Hope this helps!

**Comment by notexpected** [comment]
I printed this, it works pretty well, however I ended up discovering my TX contact did not have connectivity to the TX port, so I had to use BDM probes on the chip to do the flash.

For those looking to use this, I recommend not using screwing into the base, but rather putting a bolt thru each hole, securing with a nut, then placing the jig on the three bolts. This made it possible to check alignment with a connectivity tester :)

IMPORTANT: The Vue3 is power hungry. I discovered that it was browning out, which prevent it from connecting. I had to use two AA batteries in series (3.3v) **instead** of the VCC of my board to give it enough juice to flash

**Reply by MichaelWoodc** [reply]
> I printed this, it works pretty well, however I ended up discovering my TX contact did not have connectivity to the TX port, so I had to use BDM probes on the chip to do the flash.
> 
> For those looking to use this, I recommend not using screwing into the base, but rather putting a bolt thru each hole, securing with a nut, then placing the jig on the three bolts. This made it possible to check alignment with a connectivity tester :)
> 
> IMPORTANT: The Vue3 is power hungry. I discovered that it was browning out, which prevent it from connecting. I had to use two AA batteries in series (3.3v) **instead** of the VCC of my board to give it enough juice to flash

Appreciate the tip on the bolts, now that you mention and explain that, if I ever design another one it'll be like that!  I can attest to brownout issues here aswell.  I found two possible culprits, and not sure which one did it.

These pogo pins seem to be a hit or miss, but if you get ones with a much larger head in some other equipment, they can short connections.  I would sometimes have to sort of press the jig to the right when tightening the screws for it to make contact. 

If there's any oxidation on the pad or on the pin anyhow, simply putting them on and off a few times (especially with the sharp ones) will pierce any minor oxidation on the pads on the device and hopefully make good contact between the pin components again too.  I'll also hit these with a tiny bit of contact cleaner (my favorite is deoxit) if I continue to experience an issue but the voltages are right.
